app/components/admin/error_summary_component.rb

```diff
@@ -51,4 +55,10 @@ def errors
                   object.errors
                 end
   end
+
+  def ga4_title
+    return object.class.name.humanize if [ActiveModel::Errors, Array].include?(object.class)
+
+    "#{object.try(:new_record?) ? 'New' : 'Editing'} #{object.model_name.human.downcase.titleize}"
```

Comment:
we do this so many times you it warrants a helper method 👀  but also happy to refactor as a separate piece of work